docs/current/getting-started.md

```diff
@@ -155,6 +155,18 @@ The vaquero vagrant VM (described above) has a running etcd cluster baked in. Yo
         time="2016-12-15T21:46:28Z" level=debug msg="Successful Etcd Put for key model/current" package=storage
 
 
+## simulating IPMI reboots in the virtual environment
+In a typical deployment, reprovisioning machines and multistage boots will kick off an IPMI container to force a restart.
+Since most VMs do not include an IPMI interface, we include the option of rebooting via SSH.
+
+1. Ensure the provisioned guest VM contains a public key for authorizing SSH login (see our example cloud-config scripts for an example).
```

Comment:
Also note that you should never put private_keys in the DM. We do not provide a mechanism to put private keys on agents or get them into tasks, and leave ssh_key management to the operator to manage and vaquero will not support taking on ssh_key management